backend/src/index.ts

```diff
@@ -61,6 +61,17 @@ const session_remove = async (sessionId: string) => {
   return true;
 };
 
+const sessionIdToUserId = async (sessionId: string) => {
```

Comment:
formatting for this method seems a little off (improper indentation) - pls make sure you run this file thru prettier